admin/app/components/solidus_admin/ui/table/component.js

```diff
@@ -78,6 +78,33 @@ export default class extends Controller {
     this.render()
   }
 
+  toggleRowSelection(event) {
```

Comment:
This might need a rename now that it handles both clicks and selections, what about `rowClicked`?